app/src/main/java/com/vci/vectorcamapp/navigation/NavGraph.kt

```diff
@@ -186,16 +186,15 @@ fun NavGraph(startDestination: Destination) {
             val state by viewModel.state.collectAsStateWithLifecycle()
             ObserveAsEvents(events = viewModel.events) { event ->
                 when (event) {
-                    IncompleteSessionEvent.NavigateToSurveillanceForm -> navController.navigate(
-                        Destination.SurveillanceForm
-                    )
+                    IncompleteSessionEvent.NavigateToSurveillanceForm ->
+                        navController.navigate(Destination.SurveillanceForm)
+                    IncompleteSessionEvent.NavigateToMain -> navController.popBackStack()
                 }
             }
-            Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+            Scaffold(modifier = Modifier.fillMaxSize()) {
                 IncompleteSessionScreen(
                     state = state,
-                    onAction = viewModel::onAction,
-                    modifier = Modifier.padding(innerPadding)
+                    onAction = viewModel::onAction
```

Comment:
Add the modifier back and the inner padding, otherwise the componnents will render in places where they cannot be clicked.